### DISCUSSION TITLE
games not show up on MMP

### DISCUSSION THREAD

**Discussion by Hai1979** [OnionUI/Onion#1100] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

onion v4.2.0-rc

### Miyoo Firmware Version

202303262339

### Provide a clear and concise description of the issue

the game I downloaded does not show up on the my device after I refreshed it

### Steps to Reproduce

1. Put the roms in the card 
2. look it up on roms folder 
3. did not see it
4. refresh the page where it goes on MMP
5. did not see it

### Provide any additional context or information that may help us investigate your issue

try to reflash and put the same rom on the card it did not show up on the MMP

**Comment by Aemiii91** [comment]
Need more details:
- name of rom folder
- name of system
- file extension

**Comment by Hai1979** [comment]
GBA folder , GBA like final fantasy 6 advance #GBA.GBA

**Comment by Aemiii91** [comment]
Can you send a screenshot of the file.

**Comment by Hai1979** [comment]
![IMG_0020](https://github.com/OnionUI/Onion/assets/140565063/2d59ee25-47cc-45cd-b1e6-33d500207bf4)

**Comment by Hai1979** [comment]
here is the screen shot on my laptop

**Comment by Aemiii91** [comment]
OK, try renaming it "`Final Fantasy IV Advance.gba`"

**Comment by Hai1979** [comment]
It doesn’t work and I missing couple of games on my MMP

**Comment by Aemiii91** [comment]
How did you "refresh the page" ?

**Comment by Hai1979** [comment]
Games, Apps and then refresh the page

**Comment by Aemiii91** [comment]
What do you mean by Apps?

**Comment by Hai1979** [comment]
at the main menu screen on my MMP

**Comment by Aemiii91** [comment]
OK, then you press <kbd>SELECT</kbd> -> **Refresh all roms** ?

**Comment by Hai1979** [comment]
I press SELECT and press Refresh all Roms , and did not work at all

**Comment by Aemiii91** [comment]
Do you have an `.xml` file in your rom folder?

**Comment by Hai1979** [comment]
How do I do that?

**Comment by Hai1979** [comment]
I do have that on my card

**Comment by Aemiii91** [comment]
Why? What's it called? Try removing it.
